apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -204,6 +219,42 @@ const getEventRemindersFromQuery = (
   return eventReminders;
 };
 
+const getEventMaterialRemindersFromQuery = (
+  queryEventsContents: FetchReminderDataQuery['queryEventsContents'],
+): VideoEventReminder[] => {
+  const eventReminders = (queryEventsContents || []).reduce<
+    VideoEventReminder[]
+  >((events, event) => {
+    const videoRecordingUpdatedAt = DateTime.fromISO(
+      event.flatData.videoRecordingUpdatedAt,
+    );
+
+    const ONE_DAY_IN_SECONDS = 24 * 60 * 60;
+    const secondsFromVideoUpdatedAt =
```

Comment:
have you considered using luxon for this a little more? For instance you could use
```javascript
DateTime.now().diff(DateTime.fromISO(event.flatData.videoRecordingUpdatedAt)).seconds
  ```
to get the diff (I would imagine it would give you a positive and negative difference)